spacy_llm/tasks/lemma/task.py

```diff
@@ -0,0 +1,96 @@
+from typing import Any, Callable, Dict, Generic, Iterable, List, Optional, Type
+
+import jinja2
+from spacy.language import Language
+from spacy.scorer import Scorer
+from spacy.tokens import Doc
+from spacy.training import Example
+
+from ...ty import FewshotExample, TaskResponseParser
+from ..templates import read_template
+from ..util import ExampleType, SerializableTask
+
+DEFAULT_LEMMA_TEMPLATE_V1 = read_template("lemma")
```

Comment:
It bothers me that we didn't version the v1's in their name, can we still change that for the sake of consistency?

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/242/commits/43368c3f94f11d97f2e2efa94205a201dc8d6129.